lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -755,6 +755,12 @@ defmodule ExUnit.Assertions do
         1 + "test"
       end
 
+  To pass arguments to `function`, wrap it:
```

Comment:
In this case, we are not really passing arguments to `function`. The function still receives 0 arguments. Rather, we are showing you can invoke any code you want inside `fn -> ... end`. Is this what you had in mind?

Reply:
Of course, thank-you! :)
Yes, that is what I had in mind.

Reply:
Thanks for your time and patience Jose, you're a hero!